lib/test.js

```diff
@@ -13,13 +13,12 @@ function formatErrorValue(label, error) {
 	return {label, formatted};
 }
 
-const captureStack = start => {
+const captureStack = () => {
 	const limitBefore = Error.stackTraceLimit;
 	Error.stackTraceLimit = 1;
-	const obj = {};
-	Error.captureStackTrace(obj, start);
+	const error = new Error();
```

Comment:
Same here, I reckon we should still capture the stack trace without the `start` function.